cumulus_library_opioid/vocab/static_builder.py

```diff
@@ -0,0 +1,131 @@
+import csv
+import dataclasses
+import pathlib
+
+import pandas
+
+from cumulus_library import base_table_builder, base_utils, study_manifest
+from cumulus_library.template_sql import base_templates
+
+@dataclasses.dataclass
+class TableConfig:
+    file_path: str
+    delimiter: str
+    table_name: str
+    headers: list[str]
+    dtypes: dict
+    parquet_types: list[str]
+    filtered_name: str | None = None
+    ignore_header: bool = False
+    map_cols: list[dict] | None = None
+
+class StaticBuilder(base_table_builder.BaseTableBuilder):
```

Comment:
Can you sprinkle comments and/or spacing around this class? Code felt slightly dense while reading it - just a few newlines to separate logical blocks would help.

Reply:
ok, this is hopefully more readable now